<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="1" column="1">
     <widget class="QPushButton" name="pushButton">
      <property name="text">
       <string>Send</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0" colspan="2">
     <widget class="QTextBrowser" name="textBrowser"/>
    </item>
    <item row="2" column="1">
     <widget class="QPushButton" name="pushButton_2">
      <property name="text">
       <string>Leave Server</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0" rowspan="2">
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QLineEdit" name="lineEdit"/>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>20</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuServer">
    <property name="title">
     <string>Server</string>
    </property>
    <addaction name="actionConnect_to_Localhost"/>
   </widget>
   <addaction name="menuServer"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionConnect_to_Localhost">
   <property name="text">
    <string>Connect to Host</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
